UnitTests/MPRoktTests.m

```diff
@@ -260,4 +261,33 @@ - (void)testGetRoktPlacementAttributesMapping {
     XCTAssertEqualObjects(testResult, expectedResult, @"Mapping does not match .");
 }
 
+- (void)testSelectPlacementsIdentifyUser {
```

Comment:
Can you add a test where there is an email (foo@gmail.com) and (bar@gmail.com) is set on `selectPlacements` - then assert that identify was called with bar@gmail.com?

Reply:
I'll try. Identity is really tricky to mock well but it should be doable